docs/UserGuide.md

```diff
@@ -169,6 +169,23 @@ Examples:
 * `sort p/` returns person by ascending phone numbers `87438807`, `91031282` followed by `92492021`<br>
   ![result for 'sort p/'](images/sortByPhoneNumberResult.png)
 
+### Adding a meeting to a person: `meeting`
+
+Add a meeting to a person based on specified description and date.
+
+Format: `meeting INDEX [d/DESCRIPTION] [s/DATETIME]`
+
+* Adds a meeting to the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3,…​ and tally within range index of the displayed list.
+* At both of the fields must be provided and valid.
+* A valid `DESCRIPTION` of the meeting can only contain latin alphanumeric characters.
+* A valid `DATETIME` of the meeting can only contain valid date and 24 hour time values with a single space to separate the date and time in the format of `dd/MM/yyyy HH:mm`.
+* Duplicates meetings with the same `DESCRIPTION` and `DATETIME` in the same person is not allowed.
+
+Examples:
+* `meeting 1 d/ Meet for finals preperation s/ 12/04/2024 18:00` adds a meeting to the first person with the description of `Meet for finals preperation` and the at the date and time of `12/04/2024 18:00` 
```

Comment:
nit: Should the latter part be `and the date and time of __` without the "at the"?
Also optional and super minor to fix: `preparation` typo (if this is fixed, maybe picture also need to change)